TLM/TLM/Util/TrackUtils.cs

```diff
@@ -95,5 +104,14 @@ internal static class TrackUtils {
                 ret |= LaneEndTransitionGroup.Track;
             return ret;
         }
+
+        public static LaneEndTransitionGroup GetLaneEndTransitionGroup(this NetInfo.Lane laneInfo) {
```

Comment:
Hmm both those methods could be in API (or at least `RoutingManager`(holds the data at runtime)), since the API already contains `LaneTransitionData`, `LaneEndTransitionGroup` and `LaneEndTransitionType` definitions and this method is a helper for TransitionGroup detection, plus not strictly limited to Track group (as the name of class would suggest)

Reply:
@krzychu124 are you suggesting I expose `LaneEndTransitionGroup(*)` through IRoutingManager?
what about `ROAD/TRACK_VEHICLE/LANE_TYPES` ?

Reply:
it clarifies what is track and what is not track. but you are right. there is also `Matches()` extension that can be used in any circumstance. maybe I should call it laneInfo util? 
there is also the issue of having extensions outside extensions folder.

Reply:
@krzychu124 I renamed the class, move some stuff to extensions and exposed those 2 methods in the API.